app/controllers/application_controller.rb

```diff
@@ -1,4 +1,17 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  include Pundit::Authorization
+
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  def pundit_user
+    current_employee
+  end
+
+  private
+
+  def user_not_authorized
+    redirect_to kudos_path, notice: 'You can no longer edit or delete this kudo.'
```

Comment:
For now it is ok, but what happens if you will have more complex validations in pudit/ more reasons to fail

Reply:
Fixed.